packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -340,11 +343,71 @@ def test_vip(dcos_api_session,
                 pm_app.purge(dcos_api_session)
     assert not errors
 
+    containers = list(marathon.Container) + [Container.POD]
+    return [(container, vip_net, proxy_net)
+            for container in containers
+            for vip_net in list(marathon.Network)
+            for proxy_net in list(marathon.Network)]
+
 
-def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net, ipv6, with_port_mapping_app=False):
+@pytest.mark.parametrize('container', [marathon.Container.MESOS, Container.POD])
+@pytest.mark.parametrize('vip_net', [marathon.Network.USER])
+@pytest.mark.parametrize('proxy_net', [marathon.Network.USER])
+def test_calico_vip(dcos_api_session,
+                    container: marathon.Container,
+                    vip_net: marathon.Network,
+                    proxy_net: marathon.Network,
+                    ipv6: bool=False,
+                    with_port_mapping_app=True):
+    '''Test VIPs between the following source and destination configurations:
+        * containers: DOCKER, UCR and NONE
+        * networks: USER, BRIDGE, HOST
+        * agents: source and destnations on same agent or different agents
+        * vips: named and unnamed vip
+
+    Origin app will be deployed to the cluster with a VIP. Proxy app will be
+    deployed either to the same host or elsewhere. Finally, a thread will be
+    started on localhost (which should be a master) to submit a command to the
+    proxy container that will ping the origin container VIP and then assert
+    that the expected origin app UUID was returned
+    '''
+    if not lb_enabled():
+        pytest.skip('Load Balancer disabled')
+
+    errors = []
+    tests = setup_vip_workload_tests(dcos_api_session, container, vip_net,
+                                     proxy_net, ipv6, with_port_mapping_app, network_name="calico")
+    for vip, hosts, cmd, origin_app, proxy_app, pm_app in tests:
```

Comment:
`pm_app -> port_mapping_app` for better readability?

Reply:
Hmm, I agree with you just from this test alone, and if you see other tests they use just `pm_app`.
Anyway, I found your trouble as a maintainer of this code, so let's improve the readability as your suggestion. Thanks.